src/ansys/simai/core/api/point_cloud.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from typing import Any, Dict, List, Tuple
+
+from ansys.simai.core.api.mixin import ApiClientMixin
+
+
+class PointCloudClientMixin(ApiClientMixin):
+    def create_point_cloud(self, geometry_id: str, name: str, extension: str) -> Tuple[Dict, str]:
+        """Create an input cloud without pushing the data.
+
+        Args:
+            geometry_id: ID of the geometry to assign the input cloud to.
+            name: Name to give to the geometry.
+            extension: Extension to give to the file.
+
+        Returns:
+            Tuple with the info of the input cloud and the URL to upload ID to use for further requests.
+        """
+        post_data = {"name": name, "file_extension": extension}
+        response = self._post(f"geometries/{geometry_id}/point-cloud", json=post_data)
+        return (response["point_cloud"], response["upload_id"])
```

Comment:
returning part of the response contradicts the concept that exists in the rest of the `ApiClientMixin` classes where the full response is being return. To have a cleaner implementation, it would make sense to follow the same principle here, and let `upload_point_cloud` manage the response.